js/dom.js

```diff
@@ -0,0 +1,11 @@
+const button = document.querySelector('#menu-icon');
+
+function displayMenu() {
+  const nav = document.querySelector('nav');
+  nav.classList.toggle('sm-view');
+  const menuIcon = document.querySelector('#menu-icon');
```

Comment:
[OPTIONAL] I think you already selected this icon on top of your document `const button = document.querySelector('#menu-icon');`. You don't need to select it again if it's an element on a parent scope. You can just use the reference `button` inside the displayMenu function.